data-es5.js

```diff
@@ -1628,7 +1633,21 @@ exports.tests = [
       try { arguments.callee; return false; } catch (err) { if (!(err instanceof TypeError)) return false; }
       return true;
     */},
-    res: temp.strict,
+    res: {
+      ie10: true,
+      firefox4: true,
+      safari51: true,
+      safaritp: true,
+      webkit: true,
+      chrome13: true,
+      chrome56: false,
+      opera12: true,
+      besen: true,
+      phantom: true,
+      ejs: true,
+      ios78: true,
+      android41: true,
+    },
```

Comment:
this test is kind of obsolete because of `arguments.caller removed` test in ES2017